src/FSharp.Control.TaskSeq/Utils.fsi

```diff
@@ -42,21 +42,24 @@ module ValueTask =
     [<Obsolete "From version 0.4.0 onward, 'ValueTask.ofIValueTaskSource' is deprecated in favor of 'ValueTask.ofSource'. It will be removed in an upcoming release.">]
     val inline ofIValueTaskSource: taskSource: IValueTaskSource<bool> -> version: int16 -> ValueTask<bool>
 
-    /// Creates a ValueTask form a Task<'T>
+    /// Creates a ValueTask from a Task<'T>
     val inline ofTask: task: Task<'T> -> ValueTask<'T>
 
-    /// Ignore a ValueTask<'T>, returns a non-generic ValueTask.
-    val inline ignore: vtask: ValueTask<'T> -> ValueTask
+    /// Convert a ValueTask<'T> into a non-generic ValueTask, ignoring the result
+    val inline ignore: valueTask: ValueTask<'T> -> ValueTask
 
 module Task =
 
-    /// Convert an Async<'T> into a Task<'T>
+    /// Create a task from a value
+    val inline fromResult: value: 'U -> Task<'U>
+
+    /// Starts a running instance of an Async<'T>, represented as a Task<'T>
```

Comment:
Not entirely true (I think), as the `async` does not have to be running already. It will start the instance of the async, but it does not (quite) start a _running_ instance (as then it would already be running, right?).

Maybe this?
```suggestion
    /// Starts an instance of an Async<'T>, represented as a Task<'T>
```

Reply:
(I forgot to submit this; it was written before all the other things, but I am sending it after all your responses - You can respond if you like, but I will try to make everything self-consistent for a second round first)

detailed response under Async.toTask.
Probably best for this to be a symmetric with what ends up there.

Current proposal over there:

>/// Runs the Async<'T>computation via StartImmediateAsTask, returning the associated Task<'T>

I think it works here too

My thoughts turn to naming it something that conveys the potential starting of work aspect directly though. How about `Task.runAsync` and/or `Async.runAsTask`

Reply:
Back to `StartImmediateAsTask`: I've done a deep dive into the difference of `task { return! myAsync }` and `Async.StartImmediateAsTask myAsync`, and these methods are quite different.

The second entirely uses FSharp.Core and a bunch of extra allocations, while the first is more efficient and calls into the .NET library of task functions almost immediately. On the surface it seems that `return!` is more efficient.

Which means that we (probably) shouldn't use `StartImmediateAsTask` in the description, as it's not (entirely) correct.

As to `runAsync` etc, the idea here is to have conversion functions to/from Async as people are accustomed to conversion between like types in F#. But I'll keep it in mind as they do convey the "action" better.

Reply:
and then it did ;)
That's all fine when we are taking a running async.startchild and wanting to await it as a task. but the to/of for me is comfy and intuitive/expected with List vs Array. it gets more interesting with Array vs Seq where conversion is not even always possible. But starting a running task in the thread pool is just not the land of `of`/`to` IMO (whereas Async.ofTask and Task.toAsync is not as problematic as it's always about something that's already hot.
IIRC this all started from me wanting to get rid of a distracting `async'`, so we're probably fine to just leave it.
Also the real answer here is of course #141 / #139, but I don't have the bandwidth to really awake that bear right now
Definitely not pushing for startimmediateastask - anything starting in the new world can stick to task expr stuff

Reply:
leaving this here as the discussion thrread... I have pushed a commit that moves this on, and have noted it in another comment thread